zeppelin/cli.py

```diff
@@ -19,9 +23,23 @@ def main():
     else:
         args.out_filename = 'knowledge'
 
-    zeppelin_converter = ZeppelinConverter(args.in_filename, args.out_filename,
-                                           directory, 'anonymous', 'N/A', 'N/A')
-    zeppelin_converter.convert()
+    try:
+        with open(args.in_filename, 'rb') as raw:
+            t = json.load(raw)
+            full_path = os.path.join(directory, args.out_filename + '.md')
+            with open(full_path, 'w') as fout:
+                if 'results' in t['paragraphs'][0]:
+                    zeppelin_converter = nc(args.in_filename, args.out_filename,
+                                            directory, 'anonymous', 'N/A', 'N/A')
```

Comment:
'anonymous', 'N/A', etc. should be the default values. Please use keyword arguments in the function definition instead.